lib/core/src/lib/form/components/inplace-form-input/inplace-form-input.component.spec.ts

```diff
@@ -28,11 +27,7 @@ describe('InplaceFormInputComponent', () => {
 
     beforeEach(async () => {
         await TestBed.configureTestingModule({
-            imports: [
-                TranslateModule.forRoot(),
-                CoreTestingModule
-            ],
-            declarations: [InplaceFormInputComponent]
+            imports: [TranslateModule.forRoot(), CoreTestingModule, InplaceFormInputComponent]
```

Comment:
I think you don't need translate module here anymore, as it's already in the CoreTestingModule